web_save_discard_button/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (C) 2023-TODAY Synconics Technologies Pvt. Ltd. (<http://www.synconics.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Save & Discard Buttons",
+    "version": "17.0.1.0.0",
+    "summary": "Save & Discard Buttons",
+    "license": "AGPL-3",
+    "category": "Tools",
+    "author": "Synconics Technologies Pvt. Ltd., Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/web",
+    "maintainers": ["synconics"],
+    "depends": ["web"],
+    "data": [],
+    "images": ["static/description/main_screen.png"],
```

Comment:
This deprecated line may be removed. There are also some redundant lines due to being the default values.

Reply:
Thanks, fixed the manifest. Please check :)